src/controllers/accounts.js

```diff
@@ -177,6 +179,13 @@ function * subscribeTransfers () {
   })
 
   this.websocket.on('close', close)
+  // The client can request a notification for a given transfer id
```

Comment:
@MatthewPhinney I'm not ware of us having any documentation for the websocket protocol as of yet. Let me know if you happen to know.